<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JGraphEd</class>
 <widget class="QMainWindow" name="JGraphEd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1458</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphEd</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>908</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Tools">
     <item row="0" column="0">
      <widget class="QToolButton" name="Add_Vertex">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Node&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Add Node</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="move">
       <property name="text">
        <string>Move Node</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="changeEdgeCol">
       <property name="text">
        <string>Change Edge Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="changecolor">
       <property name="text">
        <string>Set Pen Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="changeNodeCol">
       <property name="text">
        <string>Change Node Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="DelNode">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete Node&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Delete Node</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="DelEdge">
       <property name="text">
        <string>Delete Edge</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="AddEdge">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Edge&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Add Edge</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="maxMatch">
       <property name="text">
        <string>Maximal Matching</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>45</y>
      <width>1481</width>
      <height>871</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1458</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Node"/>
    <addaction name="actionAdd_Edge"/>
    <addaction name="actionDelete_Node"/>
    <addaction name="actionDelete_Edge"/>
    <addaction name="actionMove_Node"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Pen_Color"/>
    <addaction name="actionChange_Node_Color"/>
    <addaction name="actionChange_Edge_Color"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Maximal_Matching"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionMaximum_Matching"/>
    <addaction name="actionMaximum_Clique"/>
    <addaction name="actionMaximum_Vertex_Cover"/>
    <addaction name="actionMaximum_Independent_Set"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help..</string>
   </property>
  </action>
  <action name="actionAdd_Node">
   <property name="text">
    <string>Add Node</string>
   </property>
  </action>
  <action name="actionAdd_Edge">
   <property name="text">
    <string>Add Edge</string>
   </property>
  </action>
  <action name="actionDelete_Node">
   <property name="text">
    <string>Delete Node</string>
   </property>
  </action>
  <action name="actionDelete_Edge">
   <property name="text">
    <string>Delete Edge</string>
   </property>
  </action>
  <action name="actionMove_Node">
   <property name="text">
    <string>Move Node</string>
   </property>
  </action>
  <action name="actionSet_Pen_Color">
   <property name="text">
    <string>Set Pen Color</string>
   </property>
  </action>
  <action name="actionChange_Node_Color">
   <property name="text">
    <string>Change Node Color</string>
   </property>
  </action>
  <action name="actionChange_Edge_Color">
   <property name="text">
    <string>Change Edge Color</string>
   </property>
  </action>
  <action name="actionFind_Maximal_Matching">
   <property name="text">
    <string>Find Maximal Matching</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionHelp_2">
   <property name="text">
    <string>Help...</string>
   </property>
  </action>
  <action name="actionMaximum_Matching">
   <property name="text">
    <string>Maximum Matching</string>
   </property>
  </action>
  <action name="actionMaximum_Clique">
   <property name="text">
    <string>Maximum Clique</string>
   </property>
  </action>
  <action name="actionMaximum_Vertex_Cover">
   <property name="text">
    <string>Minimum Vertex Cover</string>
   </property>
  </action>
  <action name="actionMaximum_Independent_Set">
   <property name="text">
    <string>Maximum Independent Set</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>JGraphEd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
